lab/skel_graph/traversal/_examples/scc/kosaraju/main.cpp

```diff
@@ -1,61 +1,93 @@
-//Componente tare conexe - Kosaraju - O(N+M)
+// Copyright Popescu Alexandru Gabriel <alexandru.popescu0697@gmail.com> 2018
+// Copyright Darius Neatu <neatudarius@gmail.com> 2018
+// Componente tare conexe - Kosaraju - O(n + m)
```

Comment:
Let's change this line to:
// Componente Tare Conexe (CTC) - Strongly Connected Components (SCC)
// Kosaraju - O(n + m)